### PR TITLE
Add expires_in to fragment response for access token

### PR DESCRIPTION
**Pull request by danielzeller-okta**
Primary Reviewers: @royalchan-okta @mystiberry-okta 
@federations-okta 
OKTA-94115

### CODE REVIEW

File: _source/docs/api/resources/oauth2.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ mNsaWVudF9pZCI6InVBYXVub2ZXa2FESnh1a0NGZUJ4IiwidXNlcl9pZCI6IjAwdWlkNEJ4WHc2STZUV
 qZlCT4UXH4UKhJnZ5px-ArNRqwhxXWhHJisslswjPpMkx1IgrudQIjzGYbtLFjrrg2ueiU5-YfmKuJuD6O2yPWGTsV7X6i7ABT6P-t8PRz_RNbk-U1GXW
 IEkNnEWbPqYDAm_Ofh7iW0Y8WDA5ez1jbtMvd-oXMvJLctRiACrTMLJQ2e5HkbUFxgXQ_rFPNHJbNSUBDLqdi2rg_ND64DLRlXRY7hupNsvWGo0gF4WEU
 k8IZeaLjKw8UoIs-ETEwJlAMcvkhoVVOsN5dPAaEKvbyvPC1hUGXb4uuThlwdD3ECJrtwgKqLqcWonNtiw
-&token_type=Bearer&state=waojafoawjgvbf
```
Comment (royalchan-okta) [inline _source/docs/api/resources/oauth2.md]: Shouldn't you put this in the table above as well?

**Comment by royalchan-okta** [comment]
LGTM

**Comment by mystiberry-okta** [comment]
@danielzeller-okta  Looks great. Will merge for preview next week unless instructed otherwise.

**Comment by danielzeller-okta** [comment]
Yes please merge, the code went in this afternoon.

On Jul 6, 2016, at 3:31 PM, Mysti Berry <notifications@github.com<mailto:notifications@github.com>> wrote:


@danielzeller-okta<https://github.com/danielzeller-okta> Looks great. Will merge for preview next week unless instructed otherwise.

-

**Pull request merged**